### DISCUSSION TITLE
Alphabet typing

### DISCUSSION THREAD

**Discussion by sealvessel** [monkeytypegame/monkeytype#2201] [Ideas] [answered]
Just let us type "abcdefghijklmnopqrstuvwxyz"

**Comment by unrealapex** [comment, marked as answer]
You can put the alphabet in a custom test(paste multiple times for repeats):
![image](https://user-images.githubusercontent.com/79121360/147304763-68476c53-bdcf-4107-bd23-a1ef2d284772.png)
